12-target-details.tex

```diff
@@ -301,6 +301,46 @@ \section{Neko}
 
 \section{PHP}
 \label{target-php}
+\state{NoContent}
+
+\subsection{Getting started with PHP}
+\label{target-php-getting-started}
+
+Haxe can be a powerful tool for developing PHP applications. Let's look at our first example showing the toolchain. 
+
+Create a new folder and save this class as \ic{Main.hx}.
```

Comment:
I'd remove the line above and start here with "To get started with Haxe/PHP, create a new folder..."

We might want to label the section "Getting started with Haxe/PHP" too (and do the same for other targets) to not confuse google.